express/manifest_example.json

```diff
@@ -1,25 +1,25 @@
 {
```

Comment:
What is the purpose of this file? The `express` directory seems like a strange place for it.

Reply:
It's to illustrate the layout of the manifest. I'll include it somewhere in the docs